### PR TITLE
[#50] Upgrade to MongoDB 3.0 along with co-mongo/node-mongo when stable versions are out

### PR DESCRIPTION
**Pull request by tekdel**
Hi @soygul,
It looks like mongodb native driver supports ES6 and generators by the way. This task can be done without updating co-mongo.

Api of mongodb is a little bit changed. I made some changes to use new API.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +33,7 @@
     "koa-route": "2.4.2",
     "koa-send": "3.1.0",
     "lodash": "3.10.1",
+    "mongodb": "^2.1.4",
```
Comment (soygul) [inline package.json]: Remove the caret `^` to use exact version in line with the rest of the dependencies. We update deps manually after a review and testing not to break existing stuff.

**Comment by soygul** [comment]
Jood job! I'm amazed that Mongo guys actually added promise etc. support right in the driver so fast. Tested and works nicely. Will merge after the noted point above is fixed. Thanks for the contrib again.

**Comment by soygul** [comment]
Also edit your pull-request description above and add the following line to the bottom so when I merge the PR, the issue will be automatically closed for me.

```
Closes #50
```

**Comment by tekdel** [comment]
Thank you! I removed the caret and changed description.

**Comment by soygul** [comment]
Good job! Merging and making a new release as we now have enough changes to make a new minor release.

**Pull request merged**